k8s/prometheus-operator/values.yaml

```diff
@@ -43,10 +43,10 @@ prometheusOperator:
 alertmanager:
   enabled: false
 grafana:
-  defaultDashboardsEnabled: false
+  defaultDashboardsEnabled: true
```

Comment:
We should not enable this on `master` (hence remove prior to merge), because there are many dashboards that are empty or meaningless.